cli/src/main.rs

```diff
@@ -32,6 +32,9 @@ struct Args {
     #[arg(short = 'f', long = "format", help = "The output format of the file")]
     format: Option<String>,
 
+    #[arg(short = 'r', long = "registry", help = "An URL to the registry to use")]
```

Comment:
Typo (consonant sounding "U")
```suggestion
    #[arg(short = 'r', long = "registry", help = "A URL to the registry to use")]
```

Reply:
Fixed!